src/utils/url.ts

```diff
@@ -39,7 +39,11 @@ export const openRoomUrl = (url: string) => {
 };
 
 export const openUrl = (url: string) => {
-  window.open(url, "_blank", "noopener,noreferrer");
+  if (!isExternalUrl(url)) {
+    window.open(url, "_blank", "noopener,noreferrer");
```

Comment:
nit: let's use the utility function for this

Reply:
@thecadams this is in `src/utils/url.ts`, which particular utility function were you referring to? Or didn't realise this was already in `utils/url`?

Reply:
Pretty sure this is in `openUrl`.

Reply:
This is the `openUrl` utility function, and I'm adding the check to it.

Reply:
OK. Just thinking we should pull everything into `openUrl` including the `isExternalUrl` check. If opening URLs is in one place we can easily change the systemwide policy for whether to open in new window.